inc/branding/namespace.php

```diff
@@ -17,7 +17,7 @@
  */
 function bootstrap() {
 	add_action( 'add_admin_bar_menus', __NAMESPACE__ . '\\remove_wordpress_admin_bar_item' );
-	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\enqueue_admin_scripts' );
+	add_action( 'admin_bar_init', __NAMESPACE__ . '\\enqueue_admin_scripts' );
```

Comment:
😱 picking up my bad habits!

Reply:
Also the admin bar can be turned off

Reply:
This is actually intentionally like this, as it needs to enqueue on the frontend as well. The change in here broke the Altis logo in the admin bar on the frontend.